logtape/logger.ts

```diff
@@ -425,10 +424,10 @@ export class LoggerImpl implements Logger {
     }
     if (typeof category === "string") return rootLogger.getChild(category);
     if (category.length === 0) return rootLogger;
-    return rootLogger.getChild(category as readonly [string, ...string[]]);
+    return rootLogger.getChild(category);
   }
 
-  private constructor(parent: LoggerImpl | null, category: readonly string[]) {
+  constructor(parent: LoggerImpl | null, category: CategoryList) {
```

Comment:
Why did you remove the `private` keyword?

Reply:
was playing with class inheritance. this can be rolled back

Reply:
done